data/simulations/FAH_SARS-CoV-2_spike.yml

```diff
@@ -0,0 +1,40 @@
+type: md
+title: Trajectories of the SARS-CoV-2 spike protein from Folding@Home
+description: >
+  All-atom MD simulations of the SARS-CoV-2 spike protein, simulated using Folding@Home. 
+  Simulations were run using either GROMACS or OPENMM and are stored as compressed binary XTC files. 
```

Comment:
OpenMM

Reply:
Where are the PDB files needed to interpret the XTC files? Do the XTC files contain all atoms (including solvent), just protein, protein and ions?
Which simulations were run with Gromacs and which with OpenMM?

Reply:
Where are the input files that have the provenance information for this dataset?

Reply:
Don't forget you can include links just line normal Markdown!